### ISSUE TITLE
`firebase serve` fails with false negative "Error: The default Firebase app already exists"

### ISSUE THREAD

**Issue by kossnocorp** [firebase/firebase-tools#1448]
### [REQUIRED] Environment info

**firebase-tools:** 7.0.2

**Platform:** macOS

### [REQUIRED] Test case

When imported as default:

```ts
import admin from 'firebase-admin'
admin.initializeApp()
```

...`initializeApp` throws:

```
Error: The default Firebase app already exists. This means you called initializeApp() more than once without providing an app name as the second argument. In most cases you only need to call initializeApp() once. But if you do want to initialize multiple apps, pass a second argument to initializeApp() to give each app a unique name.
  at FirebaseAppError.FirebaseError [as constructor] (node_modules/firebase-admin/lib/utils/error.js:42:28)
  at FirebaseAppError.PrefixedFirebaseError [as constructor] (node_modules/firebase-admin/lib/utils/error.js:88:28)
  at new FirebaseAppError (node_modules/firebase-admin/lib/utils/error.js:122:28)
  at FirebaseNamespaceInternals.initializeApp (node_modules/firebase-admin/lib/firebase-namespace.js:68:23)
  at FirebaseNamespace.initializeApp (node_modules/firebase-admin/lib/firebase-namespace.js:392:30)
  at Proxy.<anonymous> (node_modules/firebase-tools/src/emulator/functionsEmulatorRuntime.ts:543:31)
  at node_modules/firebase-tools/src/emulator/functionsEmulatorRuntime.ts:918:22
  at Generator.next (<anonymous>)
  at fulfilled (node_modules/firebase-tools/lib/emulator/functionsEmulatorRuntime.js:4:58)
  at <anonymous>
```

When `initializeApp` is commened out:

```ts
import admin from 'firebase-admin'
// admin.initializeApp()
```

...it throws:

```
Error: The default Firebase app does not exist. Make sure you call initializeApp() before using any of the Firebase services.
  at FirebaseAppError.FirebaseError [as constructor] (node_modules/firebase-admin/lib/utils/error.js:42:28)
  at FirebaseAppError.PrefixedFirebaseError [as constructor] (node_modules/firebase-admin/lib/utils/error.js:88:28)
  at new FirebaseAppError (node_modules/firebase-admin/lib/utils/error.js:122:28)
  at FirebaseNamespaceInternals.app (node_modules/firebase-admin/lib/firebase-namespace.js:101:19)
  at FirebaseNamespace.app (node_modules/firebase-admin/lib/firebase-namespace.js:402:30)
  at FirebaseNamespace.ensureApp (node_modules/firebase-admin/lib/firebase-namespace.js:418:24)
  at FirebaseNamespace.fn (node_modules/firebase-admin/lib/firebase-namespace.js:327:30)
  at Object../GECK/firebase/firestore/adaptor/node.ts (build/development/server/webpack:/GECK/firebase/firestore/adaptor/node.ts:16:21)
  at __webpack_require__ (build/development/server/webpack:/webpack/bootstrap:19:1)
  at Object../GECK/firebase/firestore/index.ts (build/development/server/webpack:/GECK/firebase/firestore/index.ts:1:1)
```

But it works fine when import as `*`:

```ts
import * as admin from 'firebase-admin'
admin.initializeApp()
```

### [REQUIRED] Steps to reproduce

1. Import admin as default `import admin from 'firebase-admin'`
2. Initialize the app `admin.initializeApp()`
3. Start `firebase serve`

### [REQUIRED] Expected behavior

It should simply work or throw an error with a clear explanation of what's wrong. Right now it's confusing.

### [REQUIRED] Actual behavior

It fails in a mysterious way that is extremely difficult to debug.

**Comment by samtstern** [comment]
@kossnocorp thanks for filing this.  I only pretend to know JavaScript but then I realize that I don't know the difference between `import admin from 'firebase-admin'` and `import * as admin from 'firebase-admin'` so I am hoping @abeisgoat will save me.

**Comment by samtstern** [comment]
This will be fixed by #1479 since we are going to stop auto-initializing firebase app.

**Comment by Misal-Ambasta** [comment]
I'm getting similar to the above error.

`functions: The Cloud Firestore emulator is not running, so calls to Firestore will affect production.
!  Error: The default Firebase app does not exist. Make sure you call initializeApp() before using any of the Firebase services.
    at FirebaseAppError.FirebaseError [as constructor] (C:\Users\misal\Documents\Pick Up Jobs\cloud-functions\functions\node_modules\firebase-admin\lib\utils\error.js:44:28)
    at FirebaseAppError.PrefixedFirebaseError [as constructor] (C:\Users\misal\Documents\Pick Up Jobs\cloud-functions\functions\node_modules\firebase-admin\lib\utils\error.js:90:28)
    at new FirebaseAppError (C:\Users\misal\Documents\Pick Up Jobs\cloud-functions\functions\node_modules\firebase-admin\lib\utils\error.js:125:28)
    at FirebaseNamespaceInternals.app (C:\Users\misal\Documents\Pick Up Jobs\cloud-functions\functions\node_modules\firebase-admin\lib\firebase-namespace.js:97:19)
    at FirebaseNamespace.app (C:\Users\misal\Documents\Pick Up Jobs\cloud-functions\functions\node_modules\firebase-admin\lib\firebase-namespace.js:381:30)     
    at FirebaseNamespace.ensureApp (C:\Users\misal\Documents\Pick Up Jobs\cloud-functions\functions\node_modules\firebase-admin\lib\firebase-namespace.js:395:24)
    at Proxy.fn (C:\Users\misal\Documents\Pick Up Jobs\cloud-functions\functions\node_modules\firebase-admin\lib\firebase-namespace.js:239:30)
    at Object.<anonymous> (C:\Users\misal\Documents\Pick Up Jobs\cloud-functions\functions\controller\authController.js:2:27)
    at Module._compile (internal/modules/cjs/loader.js:1133:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1153:10)
!  We were unable to load your functions code. (see above)`

My local emulators got stuck with the above error whenever I start my emulators, even I'm not able to deploy any functions, on deploying it showing the same error.
How to resolve this? @samtstern
